config/earlyapp-setup-audio.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Setup audio for Early App
+DefaultDependencies=no
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+Slice=earlyapp.slice
+
+# Load audio driver
+ExecStart=/usr/bin/modprobe -a snd-soc-sst_bxt_tdf8532
```

Comment:
Is this modeprobe works for other platforms that doesn't have snd-soc-sst_bxt_tdf8532? If not, I would like to suggest you hold this change until we come up with supporting multiple platform environments that we're working on.